.travis.yml

```diff
@@ -11,8 +12,6 @@ cache:
 language: php
 
 php:
-  - 5.4
-  - 5.5
```

Comment:
Looks like you haven't added the corresponding jobs to the matrix yet to run these on `trusty`.

Reply:
Will add those :+1: